assets/js/src/newsletters/send.jsx

```diff
@@ -74,6 +74,38 @@ const NewsletterSend = createReactClass({ // eslint-disable-line react/prefer-es
     return jQuery('#mailpoet_newsletter').parsley().isValid();
   },
 
+  isValidFromAddress: function isValidFromAddress() {
+    const fromAddress = jQuery('#field_sender_address').val();
```

Comment:
You should be able to read fromAddress from state (`this.state.item. sender_address `) which is I think a nicer way of doing it.